Jint/Runtime/StatementInterpreter.cs

```diff
@@ -19,7 +19,30 @@ public StatementInterpreter(Engine engine)
 
         private Completion ExecuteStatement(Statement statement)
         {
-            return _engine.ExecuteStatement(statement);
+            try
```

Comment:
Can we refactor all Jint's exceptions to inherit from a common one, like `JintException`, then catch this one a throw on it, and fallback to you last catch otherwise?